docker/README.md

```diff
@@ -0,0 +1,45 @@
+## Run the container
+Change to the *docker* directory of this repository:  
+`cd docker`
+   
+#### as non-root user (recommended)
+`docker-compose run detectron2`
+
+#### as root user (not recommended)
+`docker-compose run --user=root detectron2`
+
+#### with a persistent cache directory
+Prevents models to be re-downloaded on every run, by storing them in your */tmp* directory.
+
+`docker-compose run --volume=/tmp:/tmp:rw detectron2`
+
+## Rebuild the container 
+Rebuilding the container is only necessary when *Dockerfile* has been changed. The initial build is done automatically.  
+1. Change to the *docker* directory of this repository:  
+   `cd docker`
+2. Trigger the build:  
+   `docker-compose build detectron2`
+   
+## Install new dependencies
+### Persistent 
+Add the dependencies at the end of *Dockerfile*.  
+
+**Example:**  
+```
+...
+# Customization
+USER root
+RUN apt-get update && apt-get install -y \
+  nano vim emacs
+RUN pip install pandas
+USER appuser
+```
+
+### Temporary
+Use sudo (with your usual credentials) inside of the container. Changes will be lost when the container is restarted.  
```

Comment:
could you correct these instructions and the title?

Reply:
I corrected the instructions. However, I am not sure what has to be corrected about the title.